svir/calculations/process_layer.py

```diff
@@ -379,8 +379,10 @@ def transform_attribute(
             for feat in self.layer.getFeatures(request):
                 feat_id = feat.id()
                 value = transformed_dict[feat_id]
-                if type(value) not in (NULL, type(None)):
+                try:
                     value = float(value)
+                except Exception:
```

Comment:
I think is better to handle only `ValueError` exception instead of handling generic exceptions